packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -73,7 +73,9 @@ export class AtomicResultPrintableUri {
   }
 
   private getIndexOfEllipsis(parentsCount: number) {
-    return Math.min(parentsCount - 1, this.maxNumberOfParts);
+    const valuesToHide = Math.max(2, parentsCount - this.maxNumberOfParts);
+    const valuesToShowBeforeEllipsis = parentsCount - valuesToHide - 1;
+    return valuesToShowBeforeEllipsis;
```

Comment:
These three lines are exactly equivalent to:
```ts
return Math.min(this.maxNumberOfParts - 1, parentsCount - 3);
```

However, I think it's much more readable like this.

Reply:
Love this, readability over one-liner any days of the week !